nixos/modules/services/misc/dictd.nix

```diff
@@ -62,6 +62,9 @@ in
       description = "DICT.org Dictionary Server";
       wantedBy = [ "multi-user.target" ];
       environment = { LOCALE_ARCHIVE = "/run/current-system/sw/lib/locale/locale-archive"; };
+      # Work around the fact that dictd doesn't handle SIGTERM; it exits with
```

Comment:
This is very much a nit, but technically it doesn't exit, it terminates due to being killed.  ("Exit" often refers specifically to calling `_exit`, not to any way a process can terminate — see e.g.`wait(2)`.)

Reply:
Ok. Thanks for the info. Comment updated.